sdk/core/azure-core-amqp/src/models/amqp_value.cpp

```diff
@@ -146,6 +146,90 @@ namespace Azure { namespace Core { namespace Amqp { namespace Models {
     }
   } // namespace _detail
 
+  std::ostream& operator<<(std::ostream& os, AmqpValueType const value)
+  {
+
+    switch (value)
+    {
+      case AmqpValueType::Invalid:
+        os << "Invalid";
+        break;
+      case AmqpValueType::Null:
+        os << "Null";
+        break;
+      case AmqpValueType::Bool:
+        os << "Bool";
+        break;
+      case AmqpValueType::Ubyte:
+        os << "Ubyte";
+        break;
+      case AmqpValueType::Ushort:
+        os << "Ushort";
```

Comment:
BTW, we actually want the value types to be named `Ushort` and `Uint` and so on, not `UShort` and `UInt`, correct?